providers/pool.rb

```diff
@@ -188,7 +188,7 @@ def configure
     configure_application_pool(is_new_auto_start, "autoStart:#{new_resource.auto_start}")
     configure_application_pool(is_new_start_mode, "startMode:#{new_resource.start_mode}")
     configure_application_pool(new_resource.runtime_version && is_new_managed_runtime_version, "managedRuntimeVersion:v#{new_resource.runtime_version}")
-    configure_application_pool(new_resource.pipeline_mode && is_new_pipeline_mode, "managedPipelineMode:#{new_resource.pipeline_mode}.capitalize")
```

Comment:
I think if you wanted the capitalization, you'd be able to do {new_resource.pipeline_mode.capitalize}".  I missed the brace placement during previous code-review.  But since it's already being defined as an enum in the resource with the right capitalization, this should be ok.